conflowgen/posthoc_analyses/container_flow_adjustment_by_vehicle_type_analysis_report.py

```diff
@@ -74,11 +78,12 @@ def get_report_as_graph(self) -> object:
         Returns:
             The plotly figure of the Sankey diagram.
         """
-        import plotly.graph_objects as go  # pylint: disable=import-outside-toplevel
-
         initial_to_adjusted_outbound_flow = self.analysis.get_initial_to_adjusted_outbound_flow()
         initial_to_adjusted_outbound_flow_in_teu = initial_to_adjusted_outbound_flow.TEU
 
+        if (len(initial_to_adjusted_outbound_flow) + len(initial_to_adjusted_outbound_flow_in_teu)) == 0:
```

Comment:
Why did you choose len()? The data stucture (as displayed in PyCharm) is ` dict[ModeOfTransport, dict[ModeOfTransport, float]]` and len only checks for the number of key-value pairs. But the values are all zeros! So please check for the sum of the values instead of the length of the dictionairy.